Free the framebuffer memory already while stopping the VNC connection.
This will reduce the memory usage of our python italc interface.

UCS-Bug #36469
https://forge.univention.org/bugzilla/show_bug.cgi?id=36469

--- italc/lib/src/ItalcVncConnection.cpp.orig	2014-11-06 06:16:07.384349086 +0100
+++ italc/lib/src/ItalcVncConnection.cpp	2014-11-06 07:32:47.432175440 +0100
@@ -350,7 +350,11 @@
 {
 	stop();
 
-	delete [] m_frameBuffer;
+	if ( m_frameBuffer )
+	{
+		delete [] m_frameBuffer;
+		m_frameBuffer = NULL;
+	}
 }
 
 
@@ -368,6 +372,12 @@
 			qWarning( "ItalcVncConnection::stop(): thread is still running" );
 		}
 	}
+
+	if ( m_frameBuffer )
+	{
+		delete [] m_frameBuffer;
+		m_frameBuffer = NULL;
+	}
 }
 
 
